openhantek/src/hantek/hantekdsocontrol.cpp

```diff
@@ -220,7 +220,6 @@ HantekDsoControl::HantekDsoControl(USBDevice *device) : device(device) {
         break;
 
     case MODEL_DSO5200A:
-        [[clang::fallthrough]];
```

Comment:
Unfortunately only c++17 introduced officially the "[[fallthrough]]" annotation. The clang compiler will issue a warning if there is no annotation because it thinks that a "break" was forgotten. For now we'll remove it, but at some point it might need to be readded.